tools/Az/Az.psd1

```diff
@@ -31,7 +31,7 @@ Copyright = 'Microsoft Corporation. All rights reserved.'
 
 # Description of the functionality provided by this module
 Description = 'Microsoft Azure PowerShell - Cmdlets to manage resources in Azure. This module is compatible with PowerShell and Windows PowerShell.
-For more information about the Az module, please visit the following: https://learn.microsoft.com/powershell/azure/'
+For more information about the Az module, please visit the following: https://docs.microsoft.com/powershell/azure/'
```

Comment:
Need to update main branch from "docs" to "learn" to avoid overwritting generation branch